Lib/test/test_ssl.py

```diff
@@ -4073,6 +4075,20 @@ def test_compression_disabled(self):
                                    sni_name=hostname)
         self.assertIs(stats['compression'], None)
 
+    def test_legacy_server_connect(self):
+        client_context, server_context, hostname = testing_context()
+        client_context.options |= ssl.OP_LEGACY_SERVER_CONNECT
+        server_params_test(client_context, server_context,
```

Comment:
Is there a mechanism for creating a bad TLS server that doesn't support secure renegotiation indication?